packages/callhierarchy/src/browser/callhierarchy-tree/callhierarchy-tree-container.ts

```diff
@@ -15,10 +15,11 @@
  ********************************************************************************/
 
 import { interfaces, Container } from 'inversify';
-import { createTreeContainer, Tree, TreeImpl, TreeModel, TreeModelImpl, TreeWidget } from "@theia/core/lib/browser";
+import { createTreeContainer, Tree, TreeImpl, TreeModel, TreeModelImpl } from "@theia/core/lib/browser";
 import { CallHierarchyTree } from "./callhierarchy-tree";
 import { CallHierarchyTreeModel } from './callhierarchy-tree-model';
 import { CallHierarchyTreeWidget } from "./callhierarchy-tree-widget";
+import { TreeWidget } from '@theia/core/lib/browser/tree/tree-widget';
```

Comment:
This module should remain unchanged.

Reply:
@jbicker is it because of VS code rename refactoring?

Reply:
Yes. It was temporarily tree-react-widget.